tests/unit/docker/test_hub_build.py

```diff
@@ -86,3 +86,36 @@ def test_jina_version_freeze_no_jina_dependency(requirements, tmpdir):
     with open(requirements_file, 'r') as fp:
         requirements = pkg_resources.parse_requirements(fp)
         assert len(list(filter(lambda x: 'jina' in str(x), requirements))) == 0
+
+
+orig_dockerfile = """FROM test AS base
+RUN test
+
+FROM base
+RUN test"""
+
+
+revised_dockerfile = """FROM test AS base
+RUN test
+
+FROM base
+LABEL ai.jina.hub.label1="l"
+LABEL ai.jina.hub.label2="l m n"
+RUN test"""
+
+
+def test_get_revised_dockerfile(tmpdir):
```

Comment:
can you please add a test with the old standard to prove output is expected?

Reply:
Old standard?

Reply:
I mean the one currently on cookiecutter